src/Metadata/Get.php

```diff
@@ -43,7 +44,7 @@ public function __construct(
         array $paginationViaCursor = null,
         array $hydraContext = null,
         array $openapiContext = null,
-        bool|OpenApiOperation $openapi = null,
+        bool|OpenApiOperation|Webhook $openapi = null,
```

Comment:
we should probably change every operation that inherits from HttpCollection no?

Reply:
Yes, I will update it. Apologies, I just forgot.